main-app/app/scripts/controllers.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+    'use strict';
+    angular.module('Tombola.NoughtsAndCrosses')
+        .controller('MainController', function ($scope, $location, $http) {
+
+            $scope.currentGrid = [0,0,0,0,0,0,0,0,0];
```

Comment:
If you want to do this you can, but '000000000' can be *treated* as an array of char. Up to you, but I found it easier.